doc/source/cli_examples.rst

```diff
@@ -0,0 +1,257 @@
+  user.janedoe
+
+You can query the DIDs matching a certain pattern. It always requires to specify the scope in which you want to search::
+  $ rucio list-dids user.jdoe:*
+  +-------------------------------------------+--------------+
+  | SCOPE:NAME                                | [DID TYPE]   |
+  |-------------------------------------------+--------------|
+  | user.jdoe:user.jdoe.test.container.1234.1 | CONTAINER    |
+  | user.jdoe:user.jdoe.test.container.1234.2 | CONTAINER    |
+  | user.jdoe:user.jdoe.test.dataset.1        | DATASET      |
+  | user.jdoe:user.jdoe.test.dataset.2        | DATASET      |
+  | user.jdoe:test.file.1                     | FILE         |
+  | user.jdoe:test.file.2                     | FILE         |
+  | user.jdoe:test.file.3                     | FILE         |
+  |-------------------------------------------+--------------|
+
+You can filter by key/value, e.g.::
+
+  $ rucio list-dids --filter type=CONTAINER
+  +-------------------------------------------+--------------+
+  | SCOPE:NAME                                | [DID TYPE]   |
+  |-------------------------------------------+--------------|
+  | user.jdoe:user.jdoe.test.container.1234.1 | CONTAINER    |
+  | user.jdoe:user.jdoe.test.container.1234.2 | CONTAINER    |
+  |-------------------------------------------+--------------|
+
+If you want to resolve a collection (CONTAINER or DATASET) into the list of its constituents::
+
+  $ rucio list-content user.jdoe:user.jdoe.test.container.1234.1
+  +------------------------------------+--------------+
+  | SCOPE:NAME                         | [DID TYPE]   |
+  |------------------------------------+--------------|
+  | user.jdoe:user.jdoe.test.dataset.1 | DATASET      |
+  | user.jdoe:user.jdoe.test.dataset.2 | DATASET      |
+  +------------------------------------+--------------+
+
+
+
+You can resolve also the collections (CONTAINER or DATASET) into the list of files::
+  $ rucio list-content user.jdoe:user.jdoe.test.container.1234.1
+  +-----------------------+--------------------------------------+-------------+------------+----------+
+  | SCOPE:NAME            | GUID                                 | ADLER32     | FILESIZE   | EVENTS   |
+  |-----------------------+--------------------------------------+-------------+------------+----------|
+  | user.jdoe:test.file.1 | 9DF32550-D0D1-4482-9A26-0FBC46D6902A | ad:56fb0723 | 39.247 kB  |          |
+  | user.jdoe:test.file.2 | 67E8CF14-F953-45F3-B3F5-E6143F89915F | ad:e3e573b5 | 636.075 kB |          |
+  | user.jdoe:test.file.3 | 32CD7F8E-944B-4EA4-83E3-BABE48DB5751 | ad:22849380 | 641.427 kB |          |
+  +-----------------------+--------------------------------------+-------------+------------+----------+
+  Total files : 3
+  Total size : 1.316 MB:
+
+
+Rules operations
+================
+You can create a new rule like this::
+  $ rucio add-rules --account jdoe --lifetime 1209600 user.jdoe:user.jdoe.test.container.1234.1 1 "tier=1&disk=1"
```

Comment:
I would remove --account here, as it is not needed so much for normal users. Otherwise it looks like this is a required argument.